common/src/main/java/com/microsoft/identity/common/internal/commands/InteractiveTokenCommand.java

```diff
@@ -81,9 +81,4 @@ public void notify(int requestCode, int resultCode, final Intent data) {
     public boolean isEligibleForEstsTelemetry() {
         return true;
     }
-
```

Comment:
@shahzaibj why this change exactly?

Reply:
This was accidently set to true but it should be false....with that said since this interactive there is no impact where this is true or false because interactive requests don't go through the submitSilent method in Command Dispatcher